QuizApp/Scenes/LoginPage/Extensions/LoginVCExtension.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  LoginVCExtension.swift
+//  QuizApp
+//
+//  Created by Tamuna Kakhidze on 12.11.24.
+//
+
+import UIKit
+
+// MARK: - LoginViewController constants extension
+extension LoginViewController {
+    enum Sizing {
+        static let blueBackgroundTopAnchor: CGFloat = 0
+        static let blueBackgroundLeadingAnchor: CGFloat = 0
+        static let blueBackgroundTrailingAnchor: CGFloat = 0
+        static let blueBackgroundHeight: CGFloat = 433
```

Comment:
do you really need this to be always 433, try running smaller device or bigger device. it is better to have constraints which will be compatible with any device